src/connectionmanager.js

```diff
@@ -427,86 +408,11 @@ export default class ConnectionManager {
         }
 
         function ensureConnectUser(credentials) {
-
-            if (connectUser && connectUser.Id === credentials.ConnectUserId) {
-                return Promise.resolve();
-            }
-
-            else if (credentials.ConnectUserId && credentials.ConnectAccessToken) {
-
-                connectUser = null;
-
-                return getConnectUser(credentials.ConnectUserId, credentials.ConnectAccessToken).then(user => {
-
-                    onConnectUserSignIn(user);
-                    return Promise.resolve();
-
-                }, () => Promise.resolve());
-
-            } else {
-                return Promise.resolve();
-            }
-        }
-
-        function getConnectUser(userId, accessToken) {
-
-            if (!userId) {
-                throw new Error("null userId");
-            }
-            if (!accessToken) {
-                throw new Error("null accessToken");
-            }
-
-            const url = `https://connect.emby.media/service/user?id=${userId}`;
-
-            return ajax({
-                type: "GET",
-                url,
-                dataType: "json",
-                headers: {
-                    "X-Application": `${appName}/${appVersion}`,
-                    "X-Connect-UserToken": accessToken
-                }
-
-            });
+            return Promise.resolve();            
```

Comment:
We should just remove this entirely.

Reply:
removing this cause issue for me i think it check for auto login thing but for know i just restore some condition. what do you say?
https://github.com/jellyfin/jellyfin-apiclient-javascript/pull/18/commits/4ddb77aef5b27856a02b25a60a25e514d99bf3f6

Reply:
I'd rather rip it all out at the same time and remove all references to connect so we can squash all the bugs at once. I made the changes in Java and we had to get one or two more PRs in to fix issues.

Reply:
i am with you

Reply:
Done https://github.com/jellyfin/jellyfin-apiclient-javascript/pull/18/commits/75029fe06cf2315a929ebe1e87d3e4d931c35e6e

Reply:
The `onEnsureConnectUserDone` method still exists, but that's the last one.

Reply:
its done https://github.com/jellyfin/jellyfin-apiclient-javascript/pull/18/commits/8b4b31cb67ed05014b9c6a1506d3f0a5857ce38f